api/src/main/java/io/opencensus/tags/propagation/TagContextBinarySerializer.java

```diff
@@ -34,8 +34,9 @@
    *
    * @param tags the {@code TagContext} to serialize.
    * @return the on-the-wire representation of a {@code TagContext}.
+   * @throws TagContextSerializationException if the {@code TagContext} is too large to serialize.
```

Comment:
Shall we mention the 8192 bytes limit here?

Reply:
I realized that the maximum could change, so we probably shouldn't put it in the API.  We could include it in the documentation for the implementation instead.  I made a PR to remove it: #760